UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_Layout.cshtml

```diff
@@ -5,39 +5,60 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - UKHO.MaritimeSafetyInformation.Web</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/%40ukho/styles@1.3.21/dist/ukho.min.css" asp-append-version="true" />
     <link rel="stylesheet" href="~/UKHO.MaritimeSafetyInformation.Web.styles.css" asp-append-version="true" />
+    <link rel="stylesheet" href="~/css/nm_style.min.css" asp-append-version="true" />
 </head>
 <body>
-    <header>
-        <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-light bg-white border-bottom box-shadow mb-3">
-            <div class="container-fluid">
-                <a class="navbar-brand" asp-area="" asp-controller="Home" asp-action="Index">UKHO.MaritimeSafetyInformation.Web</a>
-                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target=".navbar-collapse" aria-controls="navbarSupportedContent"
+    <a class="skip-to-content-link" href="#headingLevelOne">Skip to content</a>
+    <header class="site-header">
+        <nav class="navbar navbar-expand-lg navbar-toggleable-sm navbar-light bg-white border-bottom box-shadow py-lg-0 py-2">
+            <div class="banner container-fluid justify-content-start">
+                <a class="navbar-brand" id="brandLogo" href="https://www.admiralty.co.uk/">
+                    <img class="logo-img" src="~/images/Admiralty-stacked-logo.svg" alt="Admiralty Maritime Data Solutions Home">
+                </a>
+                <div class="vertical-divider"></div>
+                <a class="site-title" id="siteTitle" asp-area="" asp-controller="Home" asp-action="Index" aria-label="Maritime Safety Information Home">Maritime Safety Information</a>
```

Comment:
Does header 'Maritime Safety Information' needs to be a clickable link (<a></a>)? Not sure if there is any specific reason?

Reply:
If user on another page, so from this link can jump to Maritime Safety Information home page. (If not needed we can remove link and will add normal text).